README.md

```diff
@@ -81,6 +81,61 @@ Use `pipenv run secret_key` to generate a value for SECRET_KEY and then save tha
 
         docker-compose up
 
+### NO Docker
+
+Follow the steps above but instead of running `docker-compose up`, do this to start the FrontEnd and Backend
+
+To Start the FrontEnd. First make sure you are in the root of the project. Then:
+
+      cd frontend
+      npm install
+      npm start
+
+You should be able to Visit the Frontend by going to [http://localhost:3000/](http://localhost:3000/). If you see the Home Page, you have won!
```

Comment:
Maybe add a note that `npm start` will try to automatically open localhost:3000 in your browser.